core/kazoo_endpoint/src/kz_endpoint.erl

```diff
@@ -1562,6 +1563,22 @@ bowout_settings('false') ->
     ,{<<"Loopback-Bowout">>, <<"true">>}
     ].
 
+
+-spec maybe_set_call_forward_fail(ccv_acc()) -> ccv_acc().
+maybe_set_call_forward_fail({_Endpoint, _Call, 'undefined', _CCVs}=Acc) ->
+    Acc;
+maybe_set_call_forward_fail({Endpoint, Call, CallFwd, CCVs}=Acc) ->
+    case kapps_config:get_is_true(?CONFIG_CAT, <<"fail_on_single_call_forward_reject">>, 'false') of
+        'true' ->
+            Fail = kapps_config:get_binary(?CONFIG_CAT, <<"fail_on_reject_val">>, <<"NORMAL_CLEARING">>),
```

Comment:
@jamesaimonetti @fenollp will do so :) also do you know about behaviour of `fail_on_single_reject` apparently if i send `NORMAL_CLEARING` only than also it disconnects the calls, even if the disconnect reason is `NO_ANSWER` or `USER_BUSY`. i have tried `^^:NORMAL_CLEARNING` that also seems to do the same, or normal termination is a part of `NORMAL_CLEARNING`, do you guys know anything about it. its not a big issue just a bit of annoyance.